scst-store/logs.md

```diff
@@ -0,0 +1,99 @@
+# Configuring and Understanding Store Logs
+
+This topic covers how to configure the Supply Chain Security Tools - Store to output particular log information, as well on how to interpret them.
+
+## Log Levels
+
+There are six log levels that the Supply Chain Security Tools - Store supports. 
+
+| Level   |            Description                      |
+| ------- | ------------------------------------------- |
+| Trace   | Output extended debugging logs              |
+| Debug   | Output standard debugging log               |
+| More    | Output more verbose informational logs      |
+| Default | Output standard informational logs          |
+| Less    | Outputs less verbose informational logs     |
+| Minimum | Outputs a minimal set of informational logs |
+
+When the Store is deployed at a specific log level, all logs of that level and lower will be outputted to the console. For example, setting the log level to `More` will output logs from `Minimal` to `More`, while `Debug` and `Trace` logs will be muted.
+
+Currently, the application logs output at these levels:
+* Minimum - The Store does not output any logs
+* Less - Outputs a single log line indicating the current log level the Metadata Store is configured to when the application starts.
+* Default - Outputs API endpoint access information.
+* Debug - Outputs API endpoint payload information, both for requests and responses.
+
+Other log levels do not output any other log information. However, they are there for future extensibility.
+
+If no log level is specified when the Store is installed, the log level is set to `default`.
+
+## API Endpoint Log Output
+
+When an API endpoint handles a request, the Store generates between two and five log lines. They are:
+1. When the endpoint receives a request, it outputs a "Processing request" line. This log line is shown at the `default` log level
+1. If the endpoint includes query or path parameters, it outputs a "Request parameters" line. This line logs the parameters being passed in the request. This line is shown at the `default` log level.
+1. If the endpoint takes in a request body, it outputs a "Request body" line. This line outputs the entire request body as a string. This line is shown at the `debug` log level.
+1. When the endpoint returns a response, it outputs a "Request response" line. This line is shown at the `default` log leve.
+1. If the endpoint returns a response body, it outputs a second "Request response" line with an extra key `payload`, and its value is set to the full response body. This line is shown at the `debug` log level
```

Comment:
```suggestion
1. If the endpoint returns a response body, it outputs a second "Request response" line with an extra key `payload`, and its value is set to the full response body. This line is shown at the `debug` log level.
```

Reply:
Inconsistent ending the sentence with a period or no period. Stick with one style.